workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -315,6 +356,8 @@ def _get_postphysics_stepper(
         else:
             self._log_info("Performing baseline simulation")
             stepper = None
+        if len(config.scale_tendencies) > 0 and stepper is not None:
+            stepper = TendencyScalingStepper(stepper, config.scale_tendencies)
```

Comment:
Is the intent to allow scaling nudging tendencies too? If not, it would be better to scope this more narrowly to the PureMLStepper.